.openshift-ci/jobs/update-support-packages/run.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -eoux pipefail
+
+gcloud_command() {
+    gcloud compute ssh --ssh-key-file="$GCP_SSH_KEY_FILE" "$GCLOUD_USER"@"$GCLOUD_INSTANCE" -- -T "$1"
+}
+
+WORKDIR=/go/src/github.com/stackrox/collector
+
+BRANCH="$(echo "$JOB_SPEC" | jq -r '.extra_refs[0].base_ref')"
+
+# shellcheck source=SCRIPTDIR/../../drivers/scripts/lib.sh
+source "${WORKDIR}/.openshift-ci/drivers/scripts/lib.sh"
+
+# Uncomment before merging
+#if ! pr_has_label "test-support-packages"; then
+#   if [ "${BRANCH}" != "master" ]; then
+#      exit 0
+#   fi
+#fi
+
+# shellcheck source=SCRIPTDIR/../../jobs/update-support-packages/env.sh
+source "${WORKDIR}/.openshift-ci/jobs/update-support-packages/env.sh"
+
+env
+
+"${WORKDIR}/.openshift-ci/jobs/integration-tests/gcloud-init.sh"
+
+SOURCE_ROOT="./collector"
+SUPPORT_PKG_SRC_ROOT="$SOURCE_ROOT/kernel-modules/support-packages"
+LICENSE_FILE="${SOURCE_ROOT}/collector/LICENSE-kernel-modules.txt"
+COLLECTOR_MODULES_BUCKET="gs://collector-modules/612dd2ee06b660e728292de9393e18c81a88f347ec52a39207c5166b5302b656"
+
+secrets_dir=/tmp/secret
+
+gcloud_command "git clone https://github.com/stackrox/collector.git --single-branch --branch=$BRANCH --depth=1"
+
+gcloud compute scp --recurse "${secrets_dir}" "${GCLOUD_USER}@${GCLOUD_INSTANCE}:${secrets_dir}"
```

Comment:
Which secrets do we need? I'm not comfortable with copying *everything* over into the VM tmp directory. I suggest that we add the secrets needed to the gcp secrets manager. See `https://github.com/stackrox/kernel-packer/blob/master/.openshift-ci/crawler/env.sh`

Reply:
I am not using secrets in the VM anymore.